build.yml

```diff
@@ -73,8 +76,12 @@ jobs:
 
       - bash: |
           set -ex
+          source activate base
+          conda install --yes pyyaml
```

Comment:
One issue I currently have is that the script relies on `pyyaml` to read the `yaml` files.
This means I have to install it here manually, because it can't be part of the env.
This also means that devs need to install that dependency first, before they can even create the environment with all the other deps...